Zajecia/src/main/java/pl/isa/javasmugglers/App.java

```diff
@@ -1,12 +1,14 @@
 package pl.isa.javasmugglers;
 
 
-public class App 
-{
-    public static void main( String[] args ){
+public class App {
+    public static void main(String[] args) {
         //testowe odpalenie aplikacji
+        System.out.println("WITAMY W MENAGERZE ZAJĘĆ");
         MainMenu mainMenu = new MainMenu();
-        System.out.println(mainMenu.getMenu());
-        mainMenu.userSelection(mainMenu.getValidInput());
+        while (true) {
```

Comment:
Trochę mnie kuje w oczy taki while (true), bez żadnych breaków, ani zmiennych, które można kontrolować. Wiem, że tam jest dalej System.exit(), który to ubija, ale i tak trochę mnie ciarki przeszły, jak to zobaczyłem... Ale nie żeby źle, tak tylko nieswojo mi z tym ;)

Reply:
Trochę to zrobiłem z myślą o tym bym mógł później łatwo wprowadzić ten system.exit(0) u chłopaków bo nie wiem co i kiedy prześlą (jeśli się wyrobią) a chciałbym by od nich też można było łatwo przerwać bądź wrócić. Na przyszłość będę unikał takiego rozwiania. Być może je zmienię na jutro ale poczekam na kod chłopaków. :)

Reply:
Poprawiłem to poprostu zmienną. Czytałem w internecie, że można też rekursywnie to załatwiać ale ten temat nie jest mi znany, więc narazie takie rozwiązanie.